src/timersMenuBar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Action, ActionPanel, Color, environment, Icon, List, MenuBarExtra, useNavigation } from "@raycast/api";
+import { time } from "console";
+import { useEffect, useState } from "react";
+import RenameView from "./RenameView";
+import CustomTimerView from "./startCustomTimer";
+import {
+  createCustomTimer,
+  deleteCustomTimer,
+  ensureCTFileExists,
+  getTimers,
+  readCustomTimers,
+  startTimer,
+  stopTimer,
+  formatTime,
+} from "./timerUtils";
+import { CustomTimer, Timer } from "./types";
+
+export default function Command() {
+  const [timers, setTimers] = useState<Timer[]>([]);
+  const [customTimers, setCustomTimers] = useState<Record<string, CustomTimer>>({});
+  const [isLoading, setIsLoading] = useState(timers.length === 0);
+
+  useEffect(() => {
+    setTimers(getTimers());
+    console.log(timers);
+    setCustomTimers(readCustomTimers());
+    setIsLoading(false);
+  }, []);
+
+  function handleTimerStop(timer: Timer) {
+    console.log("Clicked stop timer", timer.name);
+    setTimers(timers.filter((t) => t.originalFile !== timer.originalFile));
+    stopTimer(environment.supportPath + "/" + timer.originalFile);
+  }
+
+  function handleTimerStart(seconds: number, name: string) {
+    console.log("Clicked on " + name);
+    startTimer(seconds, name);
+  }
+
+  console.log(timers);
```

Comment:
Assuming this and a few other `console.log`s are left over from debugging, they're no longer needed.